v63004/inref/gtm8643.m

```diff
@@ -0,0 +1,44 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8643
+ 	set tcpdev="server$"_$j
+ 	set timeout=10
+ 	set tcpdev="server$"_$j
+ 	set portno=0
+
+ 	open tcpdev:(ZLISTEN=portno_":TCP":attach="server"):timeout:"SOCKET"
+
+ 	use $p	; principle device
+	WRITE "WRITE /listen(4):",!
+ 	use tcpdev
+ 	WRITE /listen(4)
+ 	WRITE /wait(timeout)
+	WRITE !
+
+ 	use $p	; principle device
+	WRITE "WRITE /listen(5):",!
```

Comment:
The 4, 5 and 6 second waits can all be folded into a loop to avoid code duplication.